src/websocket/app.ts

```diff
@@ -15,43 +15,64 @@
  *        console.error('problem installing DNA:', err)
  *      })
  */
-import * as msgpack from 'msgpack-lite'
+import msgpack from '@msgpack/msgpack';
 
-import { AppApi, CallZomeRequest, CallZomeResponse, AppInfoRequest, AppInfoResponse, CallZomeRequestGeneric, CallZomeResponseGeneric } from '../api/app'
-import { WsClient } from './client'
-import { catchError } from './common'
-import { Transformer, requesterTransformer, Requester } from '../api/common'
+import {
+  AppApi,
+  AppInfoRequest,
+  AppInfoResponse,
+  CallZomeRequestGeneric,
+  CallZomeResponseGeneric,
+} from '../api/app';
+import { WsClient } from './client';
+import { catchError } from './common';
+import { Transformer, requesterTransformer, Requester } from '../api/common';
 
 export class AppWebsocket implements AppApi {
-  client: WsClient
+  client: WsClient;
 
   constructor(client: WsClient) {
-    this.client = client
+    this.client = client;
   }
 
   static connect(url: string, signalCb?: Function): Promise<AppWebsocket> {
-    return WsClient.connect(url, signalCb).then(client => new AppWebsocket(client))
+    return WsClient.connect(url, signalCb).then(
+      client => new AppWebsocket(client)
+    );
   }
 
-  _requester = <ReqO, ReqI, ResI, ResO>(tag: string, transformer?: Transformer<ReqO, ReqI, ResI, ResO>) =>
+  _requester = <ReqO, ReqI, ResI, ResO>(
+    tag: string,
+    transformer?: Transformer<ReqO, ReqI, ResI, ResO>
+  ) =>
     requesterTransformer(
       req => this.client.request(req).then(catchError),
       tag,
       transformer
-    )
+    );
 
-  appInfo: Requester<AppInfoRequest, AppInfoResponse>
-    = this._requester('AppInfo')
-  callZome: Requester<CallZomeRequestGeneric<any>, CallZomeResponseGeneric<any>>
-    = this._requester('ZomeCallInvocation', callZomeTransform)
+  appInfo: Requester<AppInfoRequest, AppInfoResponse> = this._requester(
+    'AppInfo'
+  );
+  callZome: Requester<
+    CallZomeRequestGeneric<any>,
+    CallZomeResponseGeneric<any>
+  > = this._requester('ZomeCallInvocation', callZomeTransform);
 }
 
-const callZomeTransform: Transformer<CallZomeRequestGeneric<any>, CallZomeRequestGeneric<Buffer>, CallZomeResponseGeneric<Buffer>, CallZomeResponseGeneric<any>> = {
+const callZomeTransform: Transformer<
+  CallZomeRequestGeneric<any>,
+  CallZomeRequestGeneric<Buffer>,
+  CallZomeResponseGeneric<Buffer>,
+  CallZomeResponseGeneric<any>
+> = {
   input: (req: CallZomeRequestGeneric<any>): CallZomeRequestGeneric<Buffer> => {
-    req.payload = msgpack.encode(req.payload)
-    return req
+    req.payload = msgpack.encode(req.payload);
```

Comment:
so the api was identical? 
and this is the core change?

Reply:
Api is identical, yes.